core/ir/x86/instr.c

```diff
@@ -907,6 +907,17 @@ instr_is_rep_string_op(instr_t *instr)
             opc == OP_repne_cmps || opc == OP_rep_scas || opc == OP_repne_scas);
 }
 
+bool
+instr_is_floating_exten(instr_t *instr, dr_instr_category_t *type OUT)
+{
+    uint cat = instr_get_category(instr);
+    if (!TEST(DR_INSTR_CATEGORY_FP, cat))
+        return false;
+    if (type != NULL)
+        *type = cat;
+    return true;
+}
+
```

Comment:
Also change instr_is_floating() to call this one instead of the deprecated one?

Reply:
Sure, missed that